examples/lusid/instruments/upload-simple-instruments.sql

```diff
@@ -0,0 +1,32 @@
+-- ============================================================
+
+-- 1. In this query, we run an ETL process on some instruments.
+-- 2. First, we load a CSV file of instruments from Drive.
```

Comment:
It is an .xlsx file, either is fine so either update the file or the description.

Reply:
Change made.